src/main/java/duke/Task.java

```diff
@@ -0,0 +1,40 @@
+package duke;
+
+import java.time.LocalDate;
+
+class Task {
+    protected String content;
+    protected boolean isDone;
+    Task(String content) {
+        this.content = content;
+        this.isDone = false;
+    }
+
+    Task(String content, boolean isDone) {
+        this.content = content;
+        this.isDone = isDone;
+    }
+
+    void setDone() {
+        this.isDone = true;
+    }
+
+    String encoding() {
```

Comment:
```suggestion
    String encode() {
```
Would it be better to use the verb version of the word instead, as outlined in the [Code Quality: Naming](https://nus-cs2103-ay2122s1.github.io/website/book/codeQuality/nameWell/basic/nounsAndVerbsAsNames/) section of the textbook?